packages/bbui/src/Table/SelectEditRenderer.svelte

```diff
@@ -6,10 +6,11 @@
   export let onEdit
   export let allowSelectRows = false
   export let allowEditRows = false
+  export let allowSelectedOverride
```

Comment:
I think a slightly better way to handle this would just be reading it from the row itself, rather than pulling it into a separate prop. The flag already exists in the row, and we have `data` as an available prop (which is the row in this case), but it just isn't currently used here.

So we can just add `export let data` and consume it as `data.allowSelectedOverride`.